views/agency-requests/agency_requests_results.php

```diff
@@ -22,8 +22,8 @@
 $result = $db->query("SELECT  participants.participantid, participants.dateofbirth, participants.race, people.firstname, people.lastname, people.middleinit ".
                         "FROM participants ".
                         "INNER JOIN people ON participants.participantid = people.peopleid ".
-                        "WHERE LOWER(CONCAT(people.firstname, ' ' , people.lastname)) LIKE $1 ".
-                        "OR LOWER(CONCAT(people.lastname, ' ' , people.firstname)) LIKE $1",  ['%'.$searchquery.'%']);
+                        "WHERE LOWER(CONCAT(people.firstname, ' ' , people.middleinit , ' ' , people.lastname)) LIKE $1 ".
```

Comment:
If I had a name like "Johnny M Tester" and I searched "Johnny Tester", according to this nothing would be a match. I think there should be another OR statement that ignores the middle initial as well.